src/cli/node/with_node.rs

```diff
@@ -432,6 +445,9 @@ pub enum Command {
         /// Print the request that would be sent to the node and exit without doing anything
         #[structopt(long = "dry-run")]
         dry_run: bool,
+        /// Use weighted fee (instead of absolute)
+        #[structopt(long = "weighted_fee")]
+        weighted_fee: bool,
```

Comment:
This means that the syntax will be

    witnet node send --fee=1 --weighted_fee

I think that ideally the syntax should be

    witnet node send --weighted-fee=1

This can be implemented if we make both fields `Option<u64>`. There should be a way to tell `structopt` to accept "either --fee or --weighted-fee", I think it was called "conflicts_with" or something like that.